rama-http/src/layer/proxy_auth.rs

```diff
@@ -162,6 +196,10 @@ where
                     .body(Default::default())
                     .unwrap())
             }
+        } else if self.allow_anonymous {
+            ctx.insert(UserId::Anonymous);
+
```

Comment:
```suggestion
```

nit pick , but please delete this empty line

Reply:
No problem 71063ce1abd54aa1e4fd76cf48600749849a2ba4